scripts/trento-support.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+set -e
+
+readonly ARGS=("$@")
+declare -A VALID_FACILITIES=(
+    ["configuration"]="configuration"
+    ["base"]="base"
+    ["kubernetes"]="kubernetes"
+    ["all"]="all"
+)
+
+indent() { sed 's/^/  /'; }
+
+collect_trento_configuration() {
+    echo "#==[ Configuration File ]===========================#"
+    echo "# /etc/trento/installer.conf"
+    echo "$(</etc/trento/installer.conf)"    
+} &> "$OUTPUT"
+
+collect_base_system() {
+    echo "#==[ Command ]======================================#"
+    echo "# k3s --version"
+    k3s --version
+
+    echo "#==[ Command ]======================================#"
+    echo "# $(which helm) version"
+    helm version
+
+    echo "#==[ Command ]======================================#"
+    echo "# $(which helm) get hooks trento-server"
+    helm get hooks trento-server
+    
+    echo "#==[ Command ]======================================#"
+    echo "# $(which helm) get manifest trento-server"
+    helm get manifest trento-server | yq -n '[inputs]' | jq 'walk(if type == "object" then del(.data.privatekey, .data."postgresql-password", .data."postgresql-postgres-password", .secretKeyRef, ."admin-user", ."admin-password", ."SMTP_PASSWORD", ."ADMIN_USER", ."ADMIN_PASSWORD", ."SECRET_KEY_BASE") else . end)'
+    
+    echo "#==[ Command ]======================================#"
+    echo "# $(which helm) get notes trento-server"
+    helm get notes trento-server
+    
+    echo "#==[ Command ]======================================#"
+    echo "# $(which helm) get values trento-server"
+    helm get values trento-server | yq 'del(."trento-runner".privateKey, ."trento-web".adminUser)'
```

Comment:
I'm adding a dependency to yq in the spec and a check that exits if it's not present in the script. But for the other dependencies, it might be a bit trickier (explained in the other comments)